gcp_pilot/drive.py

```diff
@@ -1,16 +1,55 @@
 # Reference: https://developers.google.com/drive/api/reference/rest/v3
 
-from gcp_pilot.base import GoogleCloudPilotAPI
-
+from gcp_pilot.base import DiscoveryMixin, GoogleCloudPilotAPI
 
 class Drive(GoogleCloudPilotAPI):
     _scopes = ["https://www.googleapis.com/auth/drive"]
 
     def __init__(self, **kwargs):
-        super().__init__(
-            serviceName="drive",
-            version="v3",
-            **kwargs
-        )
+        super().__init__(serviceName="drive", version="v3", **kwargs)
+
+    @property
+    def get_service(self):
+        raise NotImplementedError()
+
+    def create(self, **kwargs):
+        return self._execute(self.get_service.create, **kwargs)
+
+    def delete(self, **kwargs):
+        return self._execute(self.get_service.delete, **kwargs)
+
+    def list(self, **kwargs):
+        return self._execute(self.get_service.list, **kwargs)
+
+    def update(self, **kwargs):
+        return self._execute(self.get_service.update, **kwargs)
+
+    def get(self, **kwargs):
+        return self._execute(self.get_service.get, **kwargs)
+
+
+class Permissions(DiscoveryMixin, Drive):
```

Comment:
Maybe `GoogleDrivePermission` to make it globally recognisable?

Reply:
That’s why I placed it inside the `drive` file so it’s accessible only via `gcp_pilot.drive.permissions` since it's scoped it looks good to me

Reply:
Yeah, but once imported, it'll look like just any random `Permissions` class that a project might have.
Not a hard objection though, other classes are also generic (e.g., `ApiKey`, `Calendar`)

Reply:
do you think it's worth adding prefix `GoogleDrive` to those classes?